src/libraries/PID/DDetectorMatches.h

```diff
@@ -46,6 +46,19 @@ class DBCALShowerMatchParams
 		}
 };
 
+class DFCALSingleHitMatchParams
+{
+ public:
+ DFCALSingleHitMatchParams(void): dEHit(0.0), dTHit(0.0), dx(0.0), dFlightTime(0.0), dFlightTimeVariance(0.0), dPathLength(0.0), dDOCAToHit(0.0){}
+  double dEHit; // energy of FCAL hit
+  double dTHit; // time of FCAL hit
+  double dx; //the distance the track traveled through the block
```

Comment:
If this is the distance into the block, shouldn't it be dz, for the sake of consistency?

Reply:
Fair enough -- although the class member for  the FCAL shower match parameters is also "dx"...

Reply:
ok, I see, it's fine to keep it as dx then